packages/x-tailwindcss/package.json

```diff
@@ -68,10 +68,10 @@
     "typescript": "~4.6.2",
     "vite": "~3.2.0",
     "vite-plugin-vue2": "~2.0.2",
-    "vue": "~2.6.14",
+    "vue": "~2.7.14",
```

Comment:
in the moment we start to use the composition API, we have to change the peer dependency to `2.7.x`?

why don't have it already changed?